packages/ui/src/lib/sidebar/elements/sidebarTabs/SidebarTabsList.stories.svelte

```diff
@@ -0,0 +1,53 @@
+<script context="module">
+	import { SidebarLeftContext } from '@ldn-viz/docs';
+
+	import SidebarTabList from './SidebarTabList.svelte';
+
+	export const meta = {
+		title: 'Ui/Components - Layout And Themes/Sidebar/elements/SidebarTabs',
+		component: SidebarTabList,
+		decorators: [() => SidebarLeftContext]
+	};
+</script>
+
+<script lang="ts">
+	import { SidebarTopContext } from '@ldn-viz/docs';
+	import { Story, Template, type DecoratorReturnType } from '@storybook/addon-svelte-csf';
+
+	import { Map as MapIcon, Square3Stack3d } from '@steeze-ui/heroicons';
+
+	//Example component passed to panel In reality these would be contained in your app
```

Comment:
```suggestion
	// Example component passed to panel.
	// In reality these would be be a component defined in your app, rather than a component imported from the UI library.
```